packages/config/config/devices/0x000c/hs-wx300.json

```diff
@@ -0,0 +1,207 @@
+{
+	"manufacturer": "HomeSeer Technologies",
+	"manufacturerId": "0x000c",
+	"label": "HS-WX300",
+	"description": "Wall Dimmer & Switch",
+	"devices": [
+		{
+			"productType": "0x4447",
+			"productId": "0x4037"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Companion Sync",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "3",
+			"label": "Bottom LED Operation",
+			"$import": "~/templates/master_template.json#led_indicator_two_options"
+		},
+		{
+			"#": "4",
+			"label": "Orientation",
+			"description": "Sets paddle load orientation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Top of Paddle turns load ON",
+					"value": 0
+				},
+				{
+					"label": "Bottom of Paddle turns load ON",
+					"value": 1
+				}
+			]
+		},
```

Comment:
I believe this parameter is the same as that one:
https://github.com/zwave-js/node-zwave-js/blob/b23ee76e6bb654ef76580fb5518f9e2e0a14dff6/packages/config/config/devices/0x000c/hs-ws200.json#L22-L24

Reply:
It does appear to be the same parameter; Looking at that master orientation template, it imports "base_enable_disable", which just has the options "enable" and "disable"; Those options don't really clarify this setting as well as the current-labels do.  I'll switch it if you'd like, it just didn't make good/clear sense when I was comparing objects/labels.  Or maybe the current orientation setting should be reworded, or a new cleaner/clearer template for "Which side of the paddle turns stuff on"?

Reply:
@blhoward2 what do you think? Knowing what is inverted and what not kinda makes sense.

Reply:
Ordinarily we just do enable/disable with inverted. It's pretty well understood that top = on is default, so this seems like added complexity. (And if you install it upside down, it's no longer the top...)

Reply:
Alrighty, I went ahead and just set this to the normal orientation inverted template for now

Reply:
Did you commit this change already? I'm not seeing it.

Reply:
That was odd, my cloud-vscode didn't want to add-to-commit my last changes; it's there now